.github/workflows/win_python_matrix_test.yml

```diff
@@ -0,0 +1,85 @@
+        gci env: | Sort Name
+        Write-Host ("")
+        Write-Host ("Setting Conda Env Vars... ")
+        Write-Host ("")
+        $env:CONDA_INSTALL = "conda install -q -y "
+        $env:ANACONDA = $env:CONDA_INSTALL + " -c anaconda "
+        $env:CONDAFORGE = $env:CONDA_INSTALL + " -c conda-forge --no-update-deps "
+        $env:USING_MINICONDA = 1
+        $env:ADDITIONAL_CF_PKGS="setuptools pip coverage sphinx_rtd_theme "
+        $env:MINICONDA_EXTRAS=""
+        $env:MINICONDA_EXTRAS="numpy scipy ipython openpyxl sympy pyodbc pyyaml networkx xlrd pandas matplotlib dill seaborn "
+        $env:ADDITIONAL_CF_PKGS=$env:ADDITIONAL_CF_PKGS + "pymysql pyro4 pint pathos " + $env:MINICONDA_EXTRAS
+        $env:ADDITIONAL_CF_PKGS=$env:ADDITIONAL_CF_PKGS + " glpk ipopt"
+        $env:EXP = $env:CONDAFORGE + $env:ADDITIONAL_CF_PKGS
+        Invoke-Expression $env:EXP
+        Write-Host ("Installing GAMS")
+        Invoke-WebRequest -Uri 'https://d37drm4t2jghv5.cloudfront.net/distributions/24.8.5/windows/windows_x64_64.exe' -OutFile 'windows_x64_64.exe'
+        Start-Process -FilePath 'windows_x64_64.exe' -ArgumentList '/SP- /VERYSILENT /NORESTART /DIR=.\gams /NOICONS'
+        $env:PATH += $(Get-Location).Path + "\gams"
+        Write-Host ("New Shell Environment: ")
+        gci env: | Sort Name
+        Write-Host ("")
+        Write-Host ("Update conda, then force it to NOT update itself again...")
+        Write-Host ("")
+        Invoke-Expression "conda config --set always_yes yes"
+        Invoke-Expression "conda config --set auto_update_conda false"
+        Write-Host ("")
+        Write-Host ("Clone model library and install PyUtilib and Pyomo...")
+        Write-Host ("") 
+        $env:CurrentWPref = $env:WarningPreference
+        $env:CurrentEPref = $env:ErrorActionPreference
+        $env:WarningPreference = "SilentlyContinue"
+        $env:ErrorActionPreference = "SilentlyContinue"
+        git --version     
+        git clone --quiet https://github.com/Pyomo/pyomo-model-libraries.git
+        git clone --quiet https://github.com/PyUtilib/pyutilib.git
+        cd pyutilib
+        python setup.py develop
+        cd ..
+        python setup.py develop
+        $env:WarningPreference = $env:CurrentWPref
+        $env:ErrorActionPreference = $env:CurrentEPref
+    - name: Install extensions
+      shell: pwsh
+      run: |
+        Write-Host "Pyomo download-extensions"
+        Invoke-Expression "pyomo download-extensions"
+        Invoke-Expression "pyomo build-extensions"
+        Write-Host "Calling solvers"
+        Invoke-Expression "glpsol -v"
+        Invoke-Expression "ipopt -v"
+    - name: Run nightly tests with test.pyomo
+      shell: pwsh
+      run: |
+        Write-Host "Setup and run nosetests"
+        $env:BUILD_DIR = $(Get-Location).Path
+        $env:EXP = "test.pyomo -v --cat='nightly' pyomo " + $env:BUILD_DIR + "\pyomo-model-libraries"
+        Invoke-Expression $env:EXP
```

Comment:
You are missing a newline at the end of this file.

Reply:
Added a newline at the end.